packages/vue-vanilla/src/cells/DateTimeCell.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <input
+    :id="cell.id + '-input'"
+    v-model="dataTime"
+    type="datetime-local"
+    :class="styles.control.input"
+    :disabled="!cell.enabled"
+    :autofocus="appliedOptions.focus"
+    :placeholder="appliedOptions.placeholder"
+    @change="onChange"
+    @focus="isFocused = true"
+    @blur="isFocused = false"
+  />
+</template>
+
+<script setup lang="ts">
+import { ref, watch } from 'vue';
+import { useJsonFormsCell } from '@jsonforms/vue';
+import type { CellProps, RankedTester } from '@jsonforms/core';
+import { isDateTimeControl, rankWith } from '@jsonforms/core';
+import { useVanillaCell } from '../util';
+
+const props = defineProps<CellProps>();
+
+const input = useVanillaCell(useJsonFormsCell(props), (target) =>
+  '' !== target.value ? toISO(target.value) : undefined
+);
+const { styles, cell, appliedOptions, onChange, isFocused } = input;
+
+const fromISO = (str: string | undefined) => str?.slice(0, 16);
+const toISO = (str: string) => str + ':00.000Z';
+
+const dataTime = ref();
+const setDateTime = (str: string | undefined) => {
+  dataTime.value = fromISO(str);
+};
+
+setDateTime(props.data.value);
+watch(() => props.data, setDateTime);
```

Comment:
This is basically just a mapper, right? So I don't think we need to go through an additional binding, couldn't we just do the following:
```
":value=fromISO(cell.data)"
```

Also `props.data` does not exist, right?

Reply:
yes, `props.data` isnt right, i have changed it to `input.cell.value.data`

but this code is required because of the `watch()` on `cell.data`.
in this cell dont use `":value=fromISO(cell.data)"`, i use `v-model="dataTime"`, that fixes the issue #2156